src/CommandHandler.h

```diff
@@ -18,6 +18,7 @@ struct StreamSettings {
   std::string Source;
   bool RunParallel = false;
   std::string ConfigStreamJson;
+  bool ReopenedOk = false;
```

Comment:
This gets set later on but is not read anywhere. Was it a 'work in progress' that got checked in by mistake?